src/plugins/intel_cpu/tests/functional/utils/cpu_test_utils.cpp

```diff
@@ -471,4 +472,74 @@ void CheckNumberOfNodesWithType(const ov::CompiledModel& compiledModel,
                                 size_t expectedCount) {
     CheckNumberOfNodesWithTypes(compiledModel, {nodeType}, expectedCount);
 }
+
+
+// deduce the actual precision of the operation given the ngraph level operation precision and the plugin config
+ov::element::Type
+CPUTestsBase::get_default_imp_precision_type(const ov::element::Type& op_prc,
+                                             const ov::AnyMap& configuration) {
+#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
+    return op_prc;
```

Comment:
NIT. Some variable names are in `s_n_a_k_e`, some in `CamelCase`. Yes, we don't have strict guidelines, but doesn't it make sense to keep the same style at least withing particular method implementations?

Reply:
updated, use **CamelCase** for variable  in this function